src/plots/cartesian/graph_interact.js

```diff
@@ -21,7 +22,7 @@ fx.layoutAttributes = {
     dragmode: {
         valType: 'enumerated',
         role: 'info',
-        values: ['zoom', 'pan', 'orbit', 'turntable'],
+        values: ['zoom', 'pan', 'select', 'lasso', 'orbit', 'turntable'],
         description: 'Determines the mode of drag interactions.'
```

Comment:
We should add info about when `select` and `lasso` have an effect in the `description` field.